.github/workflows/nodejs.yml

```
@@ -0,0 +1,25 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+name: Node.js CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: [ubuntu-latest, macos-latest, ubuntu-16.04]
+
+    strategy:
+      matrix:
+        node-version: [12.x, latest]
```

Since we support `"node": ">=10"`, what do you think of something like this:

```suggestion
        node-version: [10.x, 12.x, 14.x]
```

---

Potentially with `latest` also, but personally I think that it's best to update it when a new major comes out so that:

1) if something doesn't work on Node 15, it doesn't break all builds, but rather just breaks the PR that adds 15.x to the list
2) we don't have to either run 14.x twice (`latest, 14.x`), or stop running on 14.x when 15.x is released (`latest`)

---

Wouldn't we also do 11 and 13? Or just LTS?

---

Just LTS for now? Once Node 15 is out it won't be an LTS but adding it will prepare the project for when 16 is out.